README.md

```diff
@@ -86,6 +86,7 @@ cd build
 cmake ..
 make
 ```
+You will also need to copy the 'data' directory from the source to your build directory.
```

Comment:
Copy, or symlink... Might as well just add `ln -s ../data` to the build instructions.